index.html

```diff
@@ -7,6 +7,42 @@
     <title>Wireframe 3</title>
   </head>
   <body>
+
+  	<section class="services">
+
+  		<div>
+  			<h1>Our Services</h1>
+  			<p>Stop wasting time and money designing and managing a website that doesn't get results. Happiness guaranteed</p>
+  		</div>
+
+  		<div>
+  		
+  			<div>
+  				<h2>Graphics Design</h2>
+  				<img src="./assets/img/f-shape-1.svg" alt="graphic design">
+  				<p>Short description for the ones who look for something new. Short description for the ones who look for something new.</p>
+  				<a href="#">learn more</a>
+  			</div>
+  		
+  		
+  			<div>
+  				<h2>Website Design</h2>
+  				<img src="./assets/img/f-shape-1.svg" alt="website design">
+  				<p>Short description for the ones who look for something new. Short description for the ones who look for something new.</p>
+  				<a href="#">learn more</a>
+  			</div>
+  		
+  		
+  			<div>
+  				<h2>Digital Marketing</h2>
+  				<img src="./assets/img/f-shape-1.svg" alt="digital marketing">
+  				<p>Short description for the ones who look for something new. Short description for the ones who look for something new.</p>
+  				<a href="#">learn more</a>
+  			</div>
+
+  		</div>
+  	</section>
+
```

Comment:
this sections needs to come after the header & nav
I will merge Jacopo's PR first, then merge that one and adjust PR